libraries/chain/database.cpp

```diff
@@ -2392,7 +2367,8 @@ namespace golos { namespace chain {
                     c.last_payout = head_block_time();
                 });
 
-                push_virtual_operation(comment_payout_update_operation(comment.author, to_string(comment.permlink)));
+                push_virtual_operation(comment_payout_update_operation(comment.author, to_string(comment.permlink),
+                    total_payout_value, curator_payout_value, beneficiary_payout_value));
```

Comment:
You can calculate these values in the social_network-plugin

Reply:
@afalaleev It's a duplication of this comment: https://github.com/GolosChain/golos/pull/867#discussion_r204278768

Reply:
curator_payout_value now has wrong value, it is better to calculate it in social_network-plugin

Reply:
@afalaleev It has a wrong value because we using curation_tokens which is just 25% rathen than correct one is `25% - pay_curators(...)`.
What should we do?
We should calculate 25% and call pay_curators-like function in social_network-plugin? 
Or should send a raw non-asset value of `25% - pay_curators(...)` to the operation instead of asset type? Why it is better?